src/components/common/Loading.tsx

```diff
@@ -0,0 +1,35 @@
+import styled from '@emotion/styled';
+
+const Loading = () => {
+  return (
+    <Wrapper>
+      <Spinner />
+    </Wrapper>
+  );
+};
```

Comment:
```suggestion
const loading = (
    <Wrapper>
      <Spinner />
    </Wrapper>
  )
```

사실 컴포넌트를 항상 함수로 만들 필요는 없답니다 ㅎㅎ
어차피 로딩은 상태에 종속적이지 않기 때문에 이렇게 최초에 한 번 만들어놓고 재활용하는 방법도 있어요.

이건 개선점이라기보단 하나의 팁? 방법? 이라고 생각해주세요!

Reply:
오! 항상 함수형으로 선언하는게 익숙해서 생각해보지 못했던 포인트입니다!
알려주셔서 감사합니다. 앞으로는 컴포넌트를 만들 때 이 부분도 생각해보도록 노력하겠습니다.